### DISCUSSION TITLE
Tile Inspector does not get/set flags on 0th track sequence

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#16555] [Feature requests] [answered]
The brake flag and indestructible flag only matter on a track element's 0th sequence. On multi-tile track elements, the tile inspector should read/write to the 0th track element instead of the current track element for these properties.

For reference, the cable lift hill is a multi-tile block brake element.

**Comment by duncanspumpkin** [comment, marked as answer]
Why? The tile inspector is for overriding normal things if you want to set things correctly you shouldn't be using the tile inspector
